src/containers/Article/__tests__/Article.spec.js

```diff
@@ -38,8 +34,10 @@ describe('Article Component Test', () => {
         },
       },
     },
-    location: {
-      pathname: '/jdakdjkajda',
+    history: {
+      location: {
+        pathname: '/jdakdjkajda',
```

Comment:
@Danielshow what does `jdakdjkajda` mean?

Reply:
That is just a random word, the pathname is getting called in the component so we set it as props. I will change the name if that's what you mean.